.travis.yml

```diff
@@ -11,3 +11,8 @@ script:
   - julia -e 'Pkg.clone(pwd()); Pkg.build("TimeSeries"); Pkg.test("TimeSeries", coverage=true)'
 after_success:
   - julia -e 'cd(Pkg.dir("TimeSeries")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())'
+
+  # Documenter auto-deploy
+  - julia -e 'Pkg.add("Documenter")'
+  - julia -e 'Pkg.add("MarketData")'
+  - julia -e 'cd(Pkg.dir("MarketTechnicals", "docs")); include("make.jl")'
```

Comment:
Did you mean TimeSeries rather than MarketTechnicals?

Reply:
oh, sorry, I just copy-paste :p